assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -61,8 +61,16 @@ export const actions = {
 
 			const { dispatch } = yield Data.commonActions.getRegistry();
 			yield actions.receiveAccounts( response.accounts );
-			dispatch( STORE_NAME ).receiveProperties( response.properties );
-			dispatch( STORE_NAME ).receiveProfiles( response.profiles );
+
+			if ( response.properties.length && response.properties[ 0 ] && response.properties[ 0 ].accountId ) {
+				const accountID = response.properties[ 0 ].accountId;
+				dispatch( STORE_NAME ).receiveProperties( response.properties, { accountID } );
+			}
+
+			if ( response.properties.length && response.properties[ 0 ] && response.properties[ 0 ].id ) {
```

Comment:
I'd suggest we use `response.profiles[ 0 ].webPropertyId` here instead, to make sure it's based on the profiles retrieved. The value _should_ be the same, but I think it's a bit more accurate like that.

Reply:
@felixarntz based on a conversation with @aaemnnosttv it was felt that as we are getting the `accountID` from `response.properties[ 0 ].accountId` that in order to insure we get the correct `propertyID` we should get it from the same place ( `response.properties[ 0 ].id` ). Happy to make the change to `response.profiles[ 0 ].webPropertyId` if you think that is the correct way :)

Reply:
@ShahAaron Yeah I think it would be better to rely on the property ID via `profiles` since those with 100% certainty contain the correct property ID. We cannot assume that the first property returned is also the exact property that profiles were requested for, so this is actually problematic.

Reply:
Fixed